File: day34/lib/models/hotel.dart
class Hotel {
  final int id;
  final String image;
  final String name;
  final String description;
  final String nbetoils;
  final String adresse;
  final String prix;

  Hotel(
    this.id,
    this.image,
    this.name,
    this.description,
    this.nbetoils,
    this.adresse,
    this.prix,
  );

  factory Hotel.fromJson(Map<String, dynamic> json) {
    return new Hotel(
        json["id"],
        json["image"],
        json["name"],
        json["description"],
        json["nbetoils"],
        json["adresse"],
        json["prix"].toString());
  }
}

File: israaAdmin/app/Models/Hotel.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Hotel extends Model
{
    use HasFactory;
    protected $fillable = [
        'image',
        'name',
        'prix',
        'description',
        'nbetoils',
        'adresse',
    ];
    public function arrangements()
    {
        return $this->hasMany(Arrangement::class);
    }
}
